src/uikit/PillInputWithLoadingOptionsAndButton.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { isEqual, keys, mapValues } from 'lodash';
+import Spinner from 'react-spinkit';
+import { css } from 'emotion';
+
+import Select, { SelectOptionDropdown, DropDownOption } from '../uikit/Select';
+import PillInputWithButton from './PillInputWithButton';
+
+const LoadingSpinner = () => (
+  <Spinner
+    fadeIn="none"
+    name="circle"
+    color="#a9adc0"
+    style={{
+      width: 15,
+      height: 15,
+      marginLeft: 7,
+      position: 'absolute',
+    }}
+  />
+);
+
+class PillInputWithLoadingOptionsAndButton extends React.Component {
+  initOptionState = (props, val = { enabled: true }) => mapValues(props.options, () => val);
```

Comment:
just nit picking :P kinda wanna see `loading: false` in val

Reply:
yeah good call :)